src/ExtendedPdo.php

```diff
@@ -250,6 +250,10 @@ public static function prepareList(Array $arr = []){
 	public function selectFrom($table, $where, $type='one', $fields = "*"){
```

Comment:
Would it work better to type-hint the `$where` parameter as an `array`? And then maybe set a default value ` = []` which is an acceptable where clause.